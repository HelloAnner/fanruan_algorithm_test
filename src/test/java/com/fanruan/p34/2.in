100
1
2
3
5
7
9
10
11
11
12
13
13
16
16
17
17
18
19
19
20
24
24
25
26
26
28
30
31
31
32
32
33
33
33
33
35
35
36
37
38
42
44
44
45
46
47
47
49
50
52
53
54
54
54
56
57
61
63
66
67
67
67
67
68
69
69
71
72
74
75
76
77
78
78
79
79
80
81
81
82
84
85
85
86
89
89
90
91
91
92
92
93
93
94
96
96
98
98
98
99
200
1
1
8
9
11
11
12
12
15
19
21
21
21
22
24
25
26
27
30
30
32
32
33
35
35
37
38
39
41
42
42
45
46
46
46
48
49
49
50
50
51
53
55
55
56
57
58
58
58
59
59
62
62
63
66
66
66
67
67
67
69
69
69
70
71
72
73
74
75
76
76
78
79
79
79
81
82
82
82
84
85
85
88
89
89
89
89
90
93
96
97
97
103
103
104
104
104
105
105
105
105
105
105
106
107
110
111
112
112
112
112
113
115
117
120
120
121
121
123
123
124
124
126
131
132
132
133
133
134
134
135
135
135
136
136
137
138
139
139
139
140
141
142
143
143
144
144
144
145
146
148
151
152
153
153
154
155
156
156
157
157
157
159
160
162
164
165
166
167
167
170
172
172
172
173
175
175
175
176
176
176
178
179
179
180
180
181
183
183
185
187
187
188
189
189
191
192
195
196
197
