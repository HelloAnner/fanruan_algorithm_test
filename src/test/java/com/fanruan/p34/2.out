b57d5cf5b8ecfac2812da704ad885202